src/System.Private.Xml/src/System/Xml/Xsl/IlGen/GenerateHelper.cs

```diff
@@ -866,38 +866,6 @@ public void Call(MethodInfo meth)
             }
         }
 
-        // BUGBUG: This method is only necessary to work around VS Bug 478350
-        public void CallToken(MethodInfo meth)
-        {
-            Debug.Assert(!(_methInfo is ConstructorBuilder));
-            MethodBuilder methBldr = _methInfo as MethodBuilder;
-
-            if (methBldr != null)
-            {
-                // Using regular reflection emit, so get a token for the specified method.
-                // The token is only valid within scope of this method's body.
-                OpCode opcode = meth.IsVirtual || meth.IsAbstract ? OpCodes.Callvirt : OpCodes.Call;
-
-                TraceCall(opcode, meth);
-
-                _ilgen.Emit(opcode, ((ModuleBuilder)methBldr.Module).MetadataToken);
-
-                if (_lastSourceInfo != null)
```

Comment:
@jkotas, I already looked into that when removing it - Call already has it: https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/src/System/Xml/Xsl/IlGen/GenerateHelper.cs#L865